docs/source/loading.mdx

```diff
@@ -69,6 +70,18 @@ Specify a custom split with the `split` parameter:
 >>> c4_validation = load_dataset("allenai/c4", data_files=data_files, split="validation")
 ```
 
+## Local loading script
+
+You may have a 🤗 Datasets loading script locally on your computer. You can load this dataset by passing to [`load_dataset`] one of the following paths:
+
+- The local path to the loading script file
+- The local path to the directory containing the loading script file (only if the script file has the same name as the directory)
+
+```py
+>>> dataset = load_dataset("path/to/local/loading_script/loading_script.py", split="train")
+>>> dataset = load_dataset("path/to/local/loading_script", split="train")  # equivalently because the file has the same name as the directory
```

Comment:
```suggestion
>>> dataset = load_dataset("path/to/local/loading_script", split="train")  # equivalent because the file has the same name as the directory
```